app/src/main/java/com/example/sports/ui/theme/Color.kt

```diff
@@ -18,10 +18,67 @@ package com.example.sports.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-val Purple80 = Color(0xFFD0BCFF)
-val PurpleGrey80 = Color(0xFFCCC2DC)
-val Pink80 = Color(0xFFEFB8C8)
+val md_theme_light_primary = Color(0xFF346A22)
+val md_theme_light_onPrimary = Color(0xFFFFFFFF)
+val md_theme_light_primaryContainer = Color(0xFFB4F399)
+val md_theme_light_onPrimaryContainer = Color(0xFF042100)
+val md_theme_light_secondary = Color(0xFF54624D)
+val md_theme_light_onSecondary = Color(0xFFFFFFFF)
+val md_theme_light_secondaryContainer = Color(0xFFD8E7CC)
+val md_theme_light_onSecondaryContainer = Color(0xFF131F0E)
+val md_theme_light_tertiary = Color(0xFF006492)
+val md_theme_light_onTertiary = Color(0xFFFFFFFF)
+val md_theme_light_tertiaryContainer = Color(0xFFC9E6FF)
+val md_theme_light_onTertiaryContainer = Color(0xFF001E2F)
+val md_theme_light_error = Color(0xFFBA1A1A)
+val md_theme_light_errorContainer = Color(0xFFFFDAD6)
+val md_theme_light_onError = Color(0xFFFFFFFF)
+val md_theme_light_onErrorContainer = Color(0xFF410002)
+val md_theme_light_background = Color(0xFFFDFDF6)
+val md_theme_light_onBackground = Color(0xFF1A1C18)
+val md_theme_light_surface = Color(0xFFFDFDF6)
+val md_theme_light_onSurface = Color(0xFF1A1C18)
+val md_theme_light_surfaceVariant = Color(0xFFDFE4D7)
+val md_theme_light_onSurfaceVariant = Color(0xFF43483F)
+val md_theme_light_outline = Color(0xFF73796E)
+val md_theme_light_inverseOnSurface = Color(0xFFF1F1EA)
+val md_theme_light_inverseSurface = Color(0xFF2F312D)
+val md_theme_light_inversePrimary = Color(0xFF99D680)
+val md_theme_light_shadow = Color(0xFF000000)
```

Comment:
Are we using this resource?
I think we can delete these resources, since we are not using. 
md_theme_light_shadow
md_theme_dark_shadow